src/tests/EdFi.DataManagementService.Tests.E2E/Features/SubclassResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,37 @@
+Feature: Subclass resources validatio
+
+        Background:
+            Given this Local Education Agency
+                  | localEducationAgencyId | nameOfInstitution | localEducationAgencyCategoryDescriptor | categories                 |
+                  | 255901                 | Grand Bend ISD    | ["Independent"]                        | ["Local Education Agency"] |
+
+        @ignore
+        Scenario: Ensure client can't get create a school with the same ID as another Subclass
+             When a POST request is made to "/ed-fi/schools" with
+                  """
+                  {
+                      "schoolId": 255901,
+                      "nameOfInstitution": "School Test",
+                      "educationOrganizationCategories": [
+                          {
+                              "educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#Educator Preparation Provider"
+                          }
+                      ],
+                      "gradeLevels": [
+                          {
+                              "gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Postsecondary"
+                          }
+                      ]
+                  }
+                  """
+             Then it should respond with 409
+              And response body is
+                  """
+                  {
+                      "detail": "A natural key conflict occurred when attempting to create a new resource 'School' with a duplicate key. The duplicated columns and values are [EducationOrganizationId] (255901).",
+                      "type": "urn:ed-fi:api:conflict:natural-key",
+                      "title": "Resource Not Unique Conflict due to natural-key",
+                      "status": 409,
+                      "correlationId": null
```

Comment:
I'm seeing a slightly different response. The ODS/API may have been updated in just the last few days. Note that they say "primary key" instead of "natural key" now (which frankly I do not like). The `type` and `title` are different. And the `detail` line above is now part of the `errors` array, with a simpler statement in the `detail`.

```json
{
  "detail": "The identifying value(s) of the item are the same as another item that already exists.",
  "type": "urn:ed-fi:api:data-conflict:non-unique-identity",
  "title": "Identifying Values Are Not Unique",
  "status": 409,
  "correlationId": "fdc49d12-be7b-4e07-aa5a-432c16d7cee3",
  "errors": [
    "A primary key conflict occurred when attempting to create or update a record in the 'School' table. The duplicate key is (EducationOrganizationId) = (255901)."
  ]
}```

Reply:
Response changed